templates/main.tpl

```diff
@@ -3,34 +3,44 @@
 
     <table class="table">
         <tr>
-            <td nowrap="nowrap"><a class="btn btn-primary" href="{#url_list_domain#}">{$PALANG.pAdminMenu_list_domain}</a></td>
-            <td>{$PALANG.pMain_overview}</td>
+            <td nowrap="nowrap"><a style="text-align:left; padding-left:15px" class="btn btn-primary btn-block" href="{#url_list_domain#}"><span class="glyphicon glyphicon-th-large"
+                                                         aria-hidden="true"></span> {$PALANG.pMenu_overview}</a></td>
+            <td style="padding-top: 15px;">{$PALANG.pMain_overview}</td>
```

Comment:
Using `btn-block` and adding icons is a good idea, but unrelated to this (quite big) PR. May I ask you to submit that as a separate, small MR?